server.js

```diff
@@ -113,6 +113,11 @@ app.get('/plugins*', function (req, res) {
   res.sendFile('build/plugins.html', {root: __dirname});
 });
 
+// blm route
+app.get('/blm*', function (req, res) {
```

Comment:
the * route handler below should catch the blm.html, so you probably won't need this.